eastdetector/__init__.py

```diff
@@ -195,12 +183,24 @@ def detect(self, image: np.ndarray, params=TextDetectorParams()) -> Union[None,
         :return: An (N x 4 x 2) array, where N is the amount of boxes found and the (4 x 2) subarray holds the corner
                  points.
         """
-        if self.__closed:
-            raise RuntimeError("TextDetector: Session has already been closed.")
-        image_resized, (ratio_h, ratio_w) = _resize_image(image)
+        image_resized, (ratio_h, ratio_w) = _resize_image(image=image, max_side_len=640)
+        h, w, _ = image_resized.shape
+
+        net = cv2.dnn.readNet(self.pb_path)
```

Comment:
let's talk once again offline whether it would be possible to keep the net as a member variable of the TextDetector class